packages/db/tests/test-StatsSearch.js

```diff
@@ -0,0 +1,441 @@
+'use strict';
+
+const test = require('ava');
+const cryptoRandomString = require('crypto-random-string');
+const range = require('lodash/range');
+const { StatsSearch } = require('../dist/search/StatsSearch');
+
+const {
+  destroyLocalTestDb,
+  generateLocalTestDb,
+  GranulePgModel,
+  CollectionPgModel,
+  fakeCollectionRecordFactory,
+  fakeGranuleRecordFactory,
+  fakeProviderRecordFactory,
+  migrationDir,
+  fakePdrRecordFactory,
+  fakeExecutionRecordFactory,
+  PdrPgModel,
+  ExecutionPgModel,
+  ProviderPgModel,
+} = require('../dist');
+
+const testDbName = `collection_${cryptoRandomString({ length: 10 })}`;
+
+test.before(async (t) => {
+  const { knexAdmin, knex } = await generateLocalTestDb(
+    testDbName,
+    migrationDir
+  );
+
+  t.context.knexAdmin = knexAdmin;
+  t.context.knex = knex;
+
+  t.context.collectionPgModel = new CollectionPgModel();
+  t.context.granulePgModel = new GranulePgModel();
+  t.context.providerPgModel = new ProviderPgModel();
+  t.context.PdrPgModel = new PdrPgModel();
+  t.context.ExecutionPgModel = new ExecutionPgModel();
+
+  const statuses = ['queued', 'failed', 'completed', 'running'];
+  const errors = [{ Error: { keyword: 'UnknownError' } }, { Error: { keyword: 'CumulusMessageAdapterError' } }, { Error: { keyword: 'IngestFailure' } }, { Error: { keyword: 'CmrFailure' } }, { Error: {} }];
+  const granules = [];
+  const collections = [];
+  const executions = [];
+  const pdrs = [];
+  const providers = [];
+
+  range(20).map((num) => (
+    // collections is never aggregate queried
```

Comment:
What do you mean? It should support all types.
https://nasa.github.io/cumulus-api/unreleased/#count

Reply:
Suggest change to:

const collections = range(20).map((num) => fakeColection..